Plugins/Events/NWScript/nwnx_events.nss

```diff
@@ -729,6 +739,11 @@ const int NWNX_EVENTS_TIMING_BAR_LOCK          = 8;
 const int NWNX_EVENTS_TIMING_BAR_CUSTOM        = 10;
 */
 
+/*
+const int NWNX_EVENTS_ITEM_EVENT_DESTROY_OBJECT = 11;
+const int NWNX_EVENTS_ITEM_EVENT_DECREMENT_STACKSIZE = 16;
+*/
```

Comment:
Are these the only events Items have?

Reply:
These are the only ones I've managed to trigger. The 1.69 decompiled code also handles EVENT_TIMED_EVENT, EVENT_APPLY_EFFECT and EVENT_REMOVE_EFFECT, but at least the latter two are not handled anymore (signature changed and ApplyEffectToObject doesn't trigger anything).